index.html

```diff
@@ -2629,9 +2685,84 @@ <h2>Production and Consumption</h2>
 and consumed, including in JSON and JSON-LD.">
         <figcaption>
 Production and consumption of representations.
+See also: <a class="longdesc-link"
+href="#production-consumption-longdesc">narrative description</a>.
         </figcaption>
       </figure>
 
+      <div class="longdesc" id="production-consumption-longdesc">
+        <p>
+The upper left quadrant of the diagram contains a rectangle with dashed grey
+outline, containing two blue-outlined rectangles, one above the other, as
+follows. The upper, larger rectangle is labeled, in blue, "Core Properties",
+and contains the following <a data-cite="INFRA#maps">INFRA</a> notation:
+        </p>
+        <pre>
+«[
+  "id" → "example:123",
+  "verificationMethod" → « «[
+    "id": "did:example:123#keys-1",
+    "controller": "did:example:123",
+    "type": "Ed25519VerificationKey2018",
+    "publicKeyBase58": "H3C2AVvLMv6gmMNam3uVA"
+  ]» »,
+  "authentication" → «
+    "did:example:123#keys-1"
+  »
+]»
+        </pre>
+The lower, smaller rectangle is labeled, in blue, "Core Representation-specific
+Entries (JSON-LD)", and contains the following monospaced <a
+data-cite="INFRA#maps">INFRA</a> notation:
+        <pre>
+«[ "@context" → "https://www.w3.org/ns/did/v1" ]»
+        </pre>
+        <p>
+From the grey-outlined rectangle, three pairs of arrows extend to three
+different black-outlined rectangles, one on the upper right of the diagram, one
+in the lower right, and one in the lower left. Each pair of arrows consists of
+one blue arrow pointing from the grey-outlined rectangle to the respective
+black-outlined rectangle, labeled "produce", and one red arrow pointing in the
+reverse direction, labeled "consume". The black-outlined rectangle in the upper
+right is labeled, "application/did+cbor" and contains hexadecimal data. The
+rectangle in the lower right is labeled "application/did+json", and contains
+the following JSON:
+        </p>
+        <pre>
+{
+  "id": "did:example:123",
+  "verificationMethod": [{
+    "id": "did:example:123#keys-1",
+    "controller": "did:example:123",
+    "type": "Ed25519VerificationKey2018",
+    "publicKeyBase58": "H3C2AVvLMv6gmMNam3uVA"
+  }],
+  "authentication": [
+    "did:example:123#keys-1"
+  ]
+}
+        </pre>
+        <p>
+The rectangle in the lower left is labeled, "application/did+ld+json", and
```

Comment:
```suggestion
The rectangle in the lower left is labeled "application/did+ld+json", and
```

Reply:
Applied in 90b6928c6b3defb792e00691720bfebd5f3cafa7